### DISCUSSION TITLE
Storage for physical sound samples?

### DISCUSSION THREAD

**Discussion by mizapf** [mamedev/mame-goodies#1] [Ideas]
Is this repository also considered to be the proper location for things like floppy sound samples which we currently have in the samples folder? I'm planning to collect some new samples with an improved test environment that I set up in the recent time.

**Comment by cuavas** [comment]
I don’t want to get into putting sound samples here.  There’s a bunch of other content I’d like to gradually add, including:
* Scripts of various kinds to use with `-autoboot_script`
* Plugins
* Controller configuration files
* hiscore.dat should probably be moved here to make it clear it’s unsupported

We can’t just go copying stuff from around the web though, we need to be rigorous ensuring everything is licensed properly.  I’d like to encourage CC0-1.0 as much as possible, and nothing more restrictive than 3-clause BSD license for scripts.
